src/test/kotlin/com/terraformation/backend/seedbank/model/accession/AccessionModelCalculationsTest.kt

```diff
@@ -104,29 +105,19 @@ internal class AccessionModelCalculationsTest : AccessionModelTest() {
     }
 
     @Test
-    fun `withdrawal of estimated seed count rounds fractional remaining weight to zero`() {
+    fun `withdrawal of estimated seed count sets remaining quantity to zero`() {
       val accession =
-          accession(remaining = grams(11), subsetCount = 10000, subsetWeight = grams(3))
-              .withCalculatedValues()
-
-      val afterWithdrawal =
-          accession.addWithdrawal(withdrawal(seeds(accession.estimatedSeedCount!!), id = null))
-
-      assertEquals(grams(0), afterWithdrawal.remaining)
-      assertEquals(AccessionState.UsedUp, afterWithdrawal.state)
-    }
-
-    @Test
-    fun `withdrawal of estimated seed count rounds negative fractional remaining weight to zero`() {
-      val accession =
-          accession(remaining = grams(8), subsetCount = 1, subsetWeight = grams(3))
+          accession(
+                  remaining = kilograms(BigDecimal("0.005324")),
+                  subsetCount = 100,
+                  subsetWeight = kilograms(BigDecimal("0.000364")))
               .withCalculatedValues()
 
-      assertEquals(3, accession.estimatedSeedCount)
+      assertEquals(1463, accession.estimatedSeedCount)
```

Comment:
Is there a reason these numbers are so specific? Would something like this be sufficient? 

```
val accession =
    accession(
            remaining = kilograms(BigDecimal("0.09")),
            subsetCount = 100,
            subsetWeight = kilograms(BigDecimal("0.03")))
        .withCalculatedValues()
assertEquals(300, accession.estimatedSeedCount)
```

Reply:
Different numbers would definitely work. The numbers here are from the real accession in production that resulted in the bug report, so my thinking is that this test serves as a regression test to verify that the real accession is now fixed. Added a comment to that effect.

Reply:
Got it, thanks